### PR TITLE
basic score gathering logic

### PR DESCRIPTION
**Pull request by stkem**
math won’t stay this way, but data flow and such should.

@lawzlo @yingjieMiao

### CODE REVIEW

File: kifi-backend/modules/curator/app/com/keepit/curator/commanders/UriScoringHelper.scala [hunk 1]
```diff
@@ -1,14 +1,78 @@
...
+    (cappedPopularity / 100.0).toFloat
+  }
+
+  private def getRawPriorScores(items: Seq[SeedItem]): Seq[Float] = items.map { item =>
+    item.priorScore.getOrElse(0.0f)
+  }
+
+  private def getRawInterestScores(items: Seq[SeedItem]): Future[(Seq[Float], Seq[Float])] = {
+    val scoreTuples: Seq[Future[(Float, Float)]] = items.map { item =>
+      cortex.userUriInterest(item.userId, item.uriId).map { //to be replaced with batch call when available
```
Comment (stkem) [inline kifi-backend/modules/curator/app/com/keepit/curator/commanders/UriScoringHelper.scala:34]: @yingjieMiao let me know when this is ready

File: kifi-backend/modules/curator/app/com/keepit/curator/commanders/UriScoringHelper.scala [hunk 2]
```diff
@@ -1,14 +1,78 @@
...
+            overallOpt.map(uis => (0.5 * uis.score + 0.5) * uis.confidence).getOrElse(0.0).toFloat,
+            recentOpt.map(uis => (0.5 * uis.score + 0.5) * uis.confidence).getOrElse(0.0).toFloat
+          )
+      }
+    }
+    Future.sequence(scoreTuples).map(_.unzip)
+  }
+
+  private def getRawSocialScores(items: Seq[SeedItem]): Future[Seq[Float]] = {
+    Future.successful(items.map(_ => 0.0f)) //to be filled in by Tan
```
Comment (stkem) [inline kifi-backend/modules/curator/app/com/keepit/curator/commanders/UriScoringHelper.scala:46]: @lawzlo this is where your work will go

File: kifi-backend/modules/curator/app/com/keepit/curator/commanders/UriScoringHelper.scala (hunk 1 above)
Comment (yingjieMiao) [inline kifi-backend/modules/curator/app/com/keepit/curator/commanders/UriScoringHelper.scala:34]: this is already ready. (haven't done the scaling yet)

Comment (stkem) [inline kifi-backend/modules/curator/app/com/keepit/curator/commanders/UriScoringHelper.scala:34]: oh, yeah, I was referring to the batch endpoint mentioned in the comment

**Comment by yingjieMiao** [comment]
lgtm

**Pull request merged**